libraries/AP_NavEKF2/AP_NavEKF2_Measurements.cpp

```diff
@@ -251,7 +251,7 @@ void NavEKF2_core::readIMUData()
     } else {
         readDeltaAngle(ins.get_primary_gyro(), imuDataNew.delAng);
     }
-    imuDataNew.delAngDT = MAX(ins.get_delta_time(),1.0e-4f);
+    imuDataNew.delAngDT = MAX(ins.get_delta_angle_dt(imu_index),1.0e-4f);
```

Comment:
I'd rather modify the prototype of readDeltaAngle to be like readDeltaVelocity, thus guarantying that delAngDT and delta_angle are read at the same moment.